### PR TITLE
PLANET-4814: Convert EN Form block to WYSIWYG

### PR DESCRIPTION
**Pull request by lithrel**
Ref: https://jira.greenpeace.org/browse/PLANET-4814
Ref: https://jira.greenpeace.org/browse/PLANET-5767

This a beta version of the ENForm wysiwyg block. 

### Sidebar settings

Screenshot from 2020-12-09 12-16-54 Screenshot from 2020-12-09 12-17-27

### Switch between Signup form & Thank you message edition

Screenshot from 2020-12-09 12-12-20

## Test

- Enable the beta blocks in Planet4 > Features
- Add a new page
- Add a block _EN Form (beta)_

This block is enabled on test instance Venus https://www-dev.greenpeace.org/test-venus/en-form-beta-test/

## Checklist

- [x] Styles
  - [x] Full width
  - [x] Full width + background
  - [x] Side column style
- [ ] Form generator
  - [x] Text / Email
  - [x] Checkbox
  - [ ] Radio
  - [x] Country selector
  - [x] Position selector
- [x] Content
  - [x] Title & text
  - [x] Background
  - [x] Campaign logo
- [ ] Thank you
  - [x] Integrated edition
  - [ ] Share buttons
    - [x] Svg icons
    - [ ] Social accounts
- [ ] [jQuery dep](https://jira.greenpeace.org/browse/PLANET-5767)
  - [ ] Fields validation (email/url)
  - [x] Form fields interdependency
  - [x] Form submit
  - [ ] Form responsive layout (?)

### CODE REVIEW

File: assets/src/blocks/ENForm/ENFormInPlaceEdit.js [hunk 1]
```diff
@@ -0,0 +1,409 @@
...
+        }
+      />
+    </BlockControls>
+    <div className="form-caption">
+      {campaign_logo &&
+        <img src={ campaign_logo }
+            alt={ content_title }
+            className="campaign-logo" />
+      }
+      <RichText
```
Comment (mleray) [inline assets/src/blocks/ENForm/ENFormInPlaceEdit.js:151]: We've agreed with the design team that we should remove all formatting options from `RichText` components (more info [here](https://jira.greenpeace.org/browse/PLANET-5780)), could you please already do that with this block? 🙂

File: templates/blocks/old_covers.twig [hunk 2]
```diff
@@ -10,7 +10,7 @@
 			{% set covers_view_class = 'show-all-covers' %}
 		{% endif %}
 
-		<section class="block covers-block take-action-covers-block {{ covers_view_class }}">
+		<section class="block covers-block {{ covers_view_class }}">
```
Comment (Inwerpsel) [inline templates/blocks/old_covers.twig:13]: Is this change intended?

**Review by Inwerpsel — COMMENTED** [review]
Not sure if it's workable to split it at this point, but there's a lot of things going on in a single commit. Would be easier to review and navigate history if the increments were a bit smaller.

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [deimos](https://www-dev.greenpeace.org/test-deimos)

:watch: 2021.06.15 12:15:21

File: assets/src/styles/blocks/ENForm/components/_enform.scss [hunk 3]
```diff
@@ -146,7 +146,7 @@
       margin-top: 19px;
       line-height: 1;
 
-      button {
+      button, div.btn {
```
Comment (Inwerpsel) [inline assets/src/styles/blocks/ENForm/components/_enform.scss:149]: Is there a reason to include the `div` element? If not better to keep specificity as low as possible.

File: assets/src/styles/editorStyle.scss [hunk 4]
```diff
@@ -9,6 +9,8 @@
 @import "master-theme/assets/src/scss/base/functions";
 @import "master-theme/assets/src/scss/base/mixins";
 @import "master-theme/assets/src/scss/base/fonts";
+@import "master-theme/assets/src/scss/components/buttons";
```
Comment (Inwerpsel) [inline assets/src/styles/editorStyle.scss:12]: Master theme now has its own `editorStyle.scss` file, maybe it's better to import it there?

**Review by Inwerpsel — APPROVED** [review]
This is way too much to review in a single PR, but pretty sure that it should only affect the beta version, so I'll approve it already so we can focus on smaller parts at a time.
